src/ralph/ui/static/ui/js/app/data_center/controllers.js

```diff
@@ -28,6 +28,10 @@ angular
         };
 
         $scope.$on('edit_rack', function (event, rack) {
+            angular.forEach($scope.data_center.rack_set, function(rack) {
```

Comment:
It is ok but for future you can use shortly way

$scope.data_center.rack_set.forEach(function(){...})